x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -207,7 +207,7 @@ export function ProcessTreeNode({
 
   const showUserEscalation = user.id && user.id !== parent.user?.id;
   const interactiveSession = !!tty;
-  const sessionIcon = interactiveSession ? 'consoleApp' : 'compute';
+  const sessionIcon = interactiveSession ? 'desktop' : 'compute';
```

Comment:
for non interactive sessions I we need to change to a gear (should be in EUI)  icon: 
![image](https://user-images.githubusercontent.com/16198204/160713338-a9584298-4bbd-4a93-bf3d-c43ebc279d23.png)

Reply:
consider it done sir!